pyartcd/pyartcd/jenkins.py

```diff
@@ -221,3 +222,14 @@ def start_sync_for_ci(version: str, blocking: bool = False):
         },
         blocking=blocking
     )
+
+
+def start_microshift_sync(version: str, assembly: str, blocking: bool = False):
```

Comment:
Is there a dry_run? It might be worth it to have a dry_run

Reply:
The job has a `DRY_RUN`, but it's not needed when triggered by automation. You can always use it when starting a run by hand, but the Python interface won't need it